### PR TITLE
Procrastination

### PR DESCRIPTION
**Pull request by cpcallen**
A bunch of cleanup that I did instead of (well, actually: while) implementing permissions.  :-/

### CODE REVIEW

File: server/startup/es5.js [hunk 1]
```diff
@@ -34,9 +34,6 @@ var decodeURI = new 'decodeURI';
 var decodeURIComponent = new 'decodeURIComponent';
 var encodeURI = new 'encodeURI';
 var encodeURIComponent = new 'encodeURIComponent';
-var suspend = new 'suspend';
-var setTimeout = new 'setTimeout';
-var clearTimeout = new 'clearTimeout';
```
Comment (NeilFraser) [inline server/startup/es5.js]: Removing this should be coupled with a corresponding addition to the core, demo, and minimal cores.

File: server/interpreter.js [hunk 2]
```diff
@@ -1201,18 +1198,17 @@ Interpreter.prototype.initString = function(scope) {
...
   this.createNativeFunction('String.prototype.repeat', wrapper, false);
 };
 
 /**
  * Initialize the Boolean class.
- * @param {!Interpreter.Scope} scope Global scope.
  */
-Interpreter.prototype.initBoolean = function(scope) {
-  var thisInterpreter = this;
+Interpreter.prototype.initBoolean_ = function() {
```
Comment (NeilFraser) [inline server/interpreter.js]: Functions with a trailing underscore should be marked as @private

File: server/interpreter.js [hunk 3]
```diff
@@ -2209,17 +2196,17 @@ Interpreter.Completion = {
  * @param {Interpreter.Value} value Value to be thrown.  If message is
  *     provided a new error object is created using value as the
  *     prototype; if not it is used directly.
- * @param {string=} opt_message Message being thrown.
+ * @param {string=} message Message being thrown.
```
Comment (NeilFraser) [inline server/interpreter.js]: {string|undefined}

File: server/interpreter.js [hunk 4]
```diff
@@ -2320,13 +2307,15 @@ Interpreter.prototype.unwind = function(type, value, label) {
 
 /**
  * Class for a scope.
- * @param {Interpreter.Scope=} parentScope Inherited scope.  Defaults to null.
+ * @param {?Interpreter.Scope=} outerScope The enclosing scope ("outer
```
Comment (NeilFraser) [inline server/interpreter.js]: Is the = valid?

File: server/interpreter.js [hunk 5]
```diff
@@ -2377,12 +2366,12 @@ Interpreter.Thread.prototype.sleepUntil = function(resumeAt) {
 
 /**
  * Returns the original source code for current state.
- * @param {number=} opt_index Optional index in stack to look from.
+ * @param {number=} index Optional index in stack to look from.
```
Comment (NeilFraser) [inline server/interpreter.js]: |undefined

File: server/interpreter.js (hunk 3 above)
Comment (cpcallen) [reply server/interpreter.js]: No.

File: server/interpreter.js (hunk 5 above)
Comment (cpcallen) [reply server/interpreter.js]: No.

**Comment by NeilFraser** [comment]
LGTM

**Pull request merged**